src/library/themes/createTheme.js

```diff
@@ -28,119 +26,115 @@ const spacing_3x = pxToEm(24);
+const grayRamp = createColorRamp('gray', 'color_gray', color);
 
 export default function createTheme(
   baseColor?: Colors = 'blue',
   overrides?: { [string]: any } = {}
 ): { [string]: any } {
   /* Theme key naming convention:
    *
-   *   [property]_[target]_[variant]_[state]
+   *   [target]_[property]_[variant]_[state]
    *
-   * E.g., A key for the border-color on focused "success" buttons and inputs
-   * would look like the following.
+   * E.g., A key for the background color on disabled inputs would look like the
+   * following.
    *
-   *   borderColor_success_focus
+   *   input_backgroundColor_disabled
    *
    * Only [property] is required. Irrelevant parts are skipped.
    */
 
-  const primaries = createColorRamp(baseColor, 'color_theme', color);
+  const themeRamp = createColorRamp(baseColor, 'color_theme', color);
 
   // prettier-ignore
   return {
-    backgroundColor_disabled: color.gray_20,
-    backgroundColor_danger: color.red_60,
-    backgroundColor_danger_muted: color.red_20,
-    backgroundColor_danger_active: color.red_70,
-    backgroundColor_danger_activeMuted: color.red_30,
-    backgroundColor_danger_focus: color.red_60,
-    backgroundColor_danger_hover: color.red_50,
-    backgroundColor_success: color.green_80,
-    backgroundColor_success_muted: color.green_20,
-    backgroundColor_success_active: color.green_90,
-    backgroundColor_success_activeMuted: color.green_30,
-    backgroundColor_success_focus: color.green_80,
-    backgroundColor_success_hover: color.green_70,
-    backgroundColor_warning: color.orange_60,
-    backgroundColor_warning_muted: color.orange_20,
-    backgroundColor_warning_active: color.orange_70,
-    backgroundColor_warning_activeMuted: color.orange_30,
-    backgroundColor_warning_focus: color.orange_60,
-    backgroundColor_warning_hover: color.orange_50,
-
-    backgroundColor_input: color.white,
-
-    backgroundColor_link_focus: color.gray_30,
+    backgroundColor_active: color.gray_20,
+    backgroundColor_disabled: color.gray_30,
+    backgroundColor_focus: color.gray_10,
+    backgroundColor_hover: color.gray_10,
+    backgroundColor_danger_active: color.red_20,
+    backgroundColor_danger_focus: color.red_10,
+    backgroundColor_danger_hover: color.red_10,
+    backgroundColor_dangerPrimary: color.red_60,
+    backgroundColor_dangerPrimary_active: color.red_70,
+    backgroundColor_dangerPrimary_focus: color.red_60,
+    backgroundColor_dangerPrimary_hover: color.red_50,
+    backgroundColor_success_active: color.green_20,
+    backgroundColor_success_focus: color.green_10,
+    backgroundColor_success_hover: color.green_10,
+    backgroundColor_successPrimary: color.green_60,
+    backgroundColor_successPrimary_active: color.green_70,
+    backgroundColor_successPrimary_focus: color.green_60,
+    backgroundColor_successPrimary_hover: color.green_50,
+    backgroundColor_theme_selected: themeRamp.color_theme_10,
```

Comment:
this doesn't seem to fit the `[target]_[property]_[variant]_[state]` schema. or am I missing something?

Reply:
I updated the key to say "variation" instead of "variation" (thanks for the great suggestion!). For this example, "theme" is the variation and "selected" is the state.